### ISSUE TITLE
Any chance of updating to support new playlist URL format?

### ISSUE THREAD

**Issue by jeremymeyers** [deepakg/spotify_collage#1]
as above

**Comment by deepakg** [comment]
Could you please try a new tool I launched today instead: https://github.com/deepakg/spotify_collage_maker

Moving away from BimPy to a web-based version that uses Flask and HTML Canvas to get the job done.

**Comment by jeremymeyers** [comment]
Yup, trying and getting a flask error about running in a production environment. Do i need some other kind of wrapper or web server thing?

" * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
Usage: flask run [OPTIONS] 
Try 'flask run --help' for help.

Error: While importing 'app', an ImportError was raised."

**Comment by deepakg** [comment]
Nope, the web server is not the problem here. You are missing some dependencies. I've updated the repository's README.md with instructions on how you could install them on your machine or use Docker to run the app.
